### PR TITLE
Combine jurisdiction functional tests

### PR DESCRIPTION
**Pull request by lara-mpt**
### Change description ###

Combine jurisdiction functional tests

### CODE REVIEW

File: src/test/functional/features/jurisdiction-may-not-be-able-to-england-and-wales.feature [hunk 1]
```diff
@@ -17,9 +15,9 @@ Feature: Jurisdiction - may not be able to get a divorce in England and Wales
     When I click "Continue"
     Then the page should include "You may not be able to get a divorce in England and Wales"
 
-  @nightly
```
Comment (linusnorton) [inline src/test/functional/features/jurisdiction-may-not-be-able-to-england-and-wales.feature:20]: should these still be nightly?

File: src/test/steps/common.ts [hunk 2]
```diff
@@ -280,7 +297,7 @@ const executeUserCaseScript = async data => {
   const testUser = await iGetTheTestUser(user);
   const api = iGetTheCaseApi(testUser);
 
-  // add a delay after logging a user in because it creates and extra case that needs to be added to the ES index
+  // add a delay after logging a user in because it creates an extra case that needs to be added to the ES index
```
Comment (linusnorton) [inline src/test/steps/common.ts:300]: if this issue is resolve we might be able to remove this now

**Review by linusnorton — APPROVED** [review]
All looks fine but there are a lot of @nightly tags removed - is that okay?

**Comment by lara-mpt** [comment]
> 

@linusnorton I've combined all of the nightly tests into one separate nightly test

File: src/test/functional/features/jurisdiction-may-not-be-able-to-england-and-wales.feature (hunk 1 above)
Comment (lara-mpt) [reply src/test/functional/features/jurisdiction-may-not-be-able-to-england-and-wales.feature:20]: ah I missed that, thanks

**Pull request merged**